wordle-web/src/scripts/letter.ts

```diff
@@ -0,0 +1,32 @@
+// Class that represents a letter in the wordle word and status
+
+export enum LetterStatus {
+  NotGuessed = 0,
+  Correct,
+  Misplaced,
+  Wrong
+}
+
+export class Letter {
+  letter: string
+  status: LetterStatus = LetterStatus.NotGuessed
+
+  constructor(letter: string) {
+    this.letter = letter
+  }
+
+  public get color() {
+    switch (this.status) {
+      case LetterStatus.NotGuessed:
```

Comment:
Redundant with the default.

Reply:
Yeah this bit was mirrored from what we went over in class so I forgot to remove that part when we ended up adding a default.

Reply:
I fixed this. It should be fine now.